Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml

```
@@ -0,0 +1,15 @@
+- type: entity
+  parent: LockerChiefJustice
+  id: LockerChiefJusticeFilled
+  name: chief justice's locker
+  components:
+  - type: StorageFill
+    contents:
+    - id: ClothingHeadsetCJ
+    - id: ClothingNeckCloakCJ
+    - id: BoxPDAJustice
+    - id: BoxEncryptionKeyJustice
+    - id: CJIDCard
+    - id: DoorRemoteJustice
+    - id: Gavel
+    - id: RubberStampCJ
```

CJ? From hit video-game GTA San Andreas?
This doesn't follow the naming convention for stamps
```suggestion
    - id: RubberStampChiefJustice

```
also a line at the end for readability